Content/Items/Gear/Weapons/Sword.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System.Linq;
+using Terraria.ID;
+
+namespace FunnyExperience.Content.Items.Gear.Weapons
+{
+	internal class Sword : Gear
+	{
+		public override string Texture => $"{FunnyExperience.ModName}/Assets/Items/Gear/Weapon/Sword/Base";
+	
+		public override void SetDefaults()
+		{
+			Item.damage = 10;
+			Item.width = 40; 
+			Item.height = 40;
+			Item.useStyle = ItemUseStyleID.Swing; 
+			Item.useTime = 20; 
+			Item.useAnimation = 20;
+			Item.autoReuse = true;
+			Item.DamageType = DamageClass.Melee;
+			Item.knockBack = 6;
+			Item.crit = 6;
+			Item.UseSound = SoundID.Item1; 
+		}
+		
+		public override void PostRoll()
+		{
+			//These don't work.... Whai?
```

Comment:
Most likely SetDefaults is called afterwards and is also called on all relevant clones. This game does a very large amount of cloning of items (and one of the worst cases of this is tooltips, it clones a new copy of the item to render every single frame! its horrifying!)